<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MSI" name="io.duotech.tests.AllOrdersTests" tests="3" failures="1" timestamp="2020-03-26T19:29:20 EDT" time="28.044" errors="0">
  <testcase name="verifyMessageAfterDeleteAll" time="9.440" classname="io.duotech.tests.AllOrdersTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Title is not correct did not expect to find [true] but found [false],
	Welcome message was wrong did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Title is not correct did not expect to find [true] but found [false],
Welcome message was wrong did not expect to find [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at io.duotech.tests.AllOrdersTests.verifyMessageAfterDeleteAll(AllOrdersTests.java:97)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyMessageAfterDeleteAll -->
  <testcase name="verifyCheckAllButton" time="10.550" classname="io.duotech.tests.AllOrdersTests"/>
  <testcase name="verifyUncheckAllButton" time="8.054" classname="io.duotech.tests.AllOrdersTests"/>
</testsuite> <!-- io.duotech.tests.AllOrdersTests -->
